docs/versioning.md

```diff
@@ -0,0 +1,26 @@
+# Supported Versions and Releases
+
+This document provides detailed information about the different versions that are supported and released for prebuilt detection rules.
+
+## Current Version
+
+The current version of prebuilt detection rules is `v8.14`.
+
+## Previous Versions Released
+
+The following version(s) are released along with the current version.
+
+- `v8.13`
+- `v8.12`
+- `v8.11`
+
+### Previous Versions Maintained
+
+The following version(s) are maintained along with the current version.
+
+- `v8.10`
+- `v8.9`
```

Comment:
If we never want to update this again, I'd make this more conceptual.

Reply:
To keep it simple to use and also use the Prep branch step to edit and reflect the right versions, this was kept non conceptual.